include/nrm.h

```diff
@@ -73,8 +73,23 @@ int nrm_finalize(void);
 #define NRM_LOG_INFO 4
 #define NRM_LOG_DEBUG 5
 
+/**
+ * Initializes NRM logging
+ *
+ * @param f: file descriptor
+ * @param nm: prefixed name for output
+ * @return 0 if successful, an error code otherwise
+ */
 int nrm_log_init(FILE *f, const char *nm);
 
+/**
+ * Prints an NRM log message at a log level.
+ *
+ * @param level: log level constant
+ * @param file: file descriptor
+ * @param line: file descriptor
+ * @param format: file descriptor
+ */
```

Comment:
don't use "descriptor" here, it means something very specific in C. Instead explain that it's info to indicate the file and line of code the message comes from, and should normally be used with `__FILE__` and `__LINE__`. format is standard printf format string.